test/test-applications/integrations/TestApplication.Wcf.Client.DotNet/StatusRequest.cs

```diff
@@ -18,7 +18,7 @@
 
 namespace TestApplication.Wcf.Client.DotNet;
 
-[DataContract]
+[DataContract(Namespace = "http://opentelemetry.io/")]
```

Comment:
Currently, contract types are not shared. This namespace override is needed in order for data contract types to be equivalent.